### ISSUE TITLE
docs/introduction/about-plugins.md: Invalid RegExp in example

### ISSUE THREAD

**Issue by felixrabe** [fuse-box/fuse-box#1260]
https://github.com/fuse-box/fuse-box/blob/70e0ce8a3c20b8e09b438885b2a82e5f033bfee1/docs/introduction/about-plugins.md, where it says:

***

For example, the code snippet above is equivalent to:

```js
plugins: [
    [ /mypath\/**.css/, CSSPlugin() ]
]
```

***

contains an invalid RegExp:

```
$ node
> /mypath\/**.css/
SyntaxError: Invalid regular expression: /mypath\/**.css/: Nothing to repeat
```

**Comment by felixrabe** [comment]
Closed #1252 (PR) in favor of creating this issue.

**Comment by nchanged** [comment]
@felixrabe 

```
"mypath**.css"
```
whereas this expression is a string

**Comment by Kompwu** [comment]
@felixrabe How's going ?

**Comment by nchanged** [comment]
I think we can close it

**Comment by felixrabe** [comment]
Actually no, because the [example](https://github.com/fuse-box/fuse-box/blob/master/docs/introduction/about-plugins.md) is still broken. ` /mypath\/**.css/` is still an invalid RegExp.

I've closed #1252 with the intention of leaving it to the project authors to decide on how to re-write the example correctly.

**Comment by nchanged** [comment]
It's not valid unless it's a simplified regexp but even then we don't need to escape slashes
